src/acceptance-test/java/org/zalando/nakadi/webservice/EventStreamReadingAT.java

```diff
@@ -315,20 +326,22 @@ public void whenInvalidCursorsThenPreconditionFailed() {
                 .body("detail", equalTo("non existing partition very_wrong_partition"));
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = 30000)
```

Comment:
Probably it was an investigation of problem, and I forgot to change it back. Will change it back.